src/protojson/ResponseMsg.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_PROTOJSON_RESPONSEMSG_H
+#define LSST_QSERV_PROTOJSON_RESPONSEMSG_H
+
+// System headers
+#include <chrono>
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+
+// Third party headers
+#include "nlohmann/json.hpp"
+
+// qserv headers
+#include "global/clock_defs.h"
+#include "global/intTypes.h"
+
+namespace lsst::qserv::qdisp {
+class UberJob;
+}
+
+namespace lsst::qserv::wbase {
+class UberJobData;
+}
+
+// This header declarations
+namespace lsst::qserv::protojson {
+
+/// This class handles the message used for most success/fail responses.
+class ResponseMsg {
+public:
+    using Ptr = std::shared_ptr<ResponseMsg>;
+
+    ResponseMsg(bool success_, std::string const& errorType_ = std::string("none"),
```

Comment:
Could be just:
```
= "none"
```
This will create a temporary string to be passed into the c-tor. And your constructor always copies that string into the data member:
```
std::string errorType;
```
So, you'd be safe here.